### ISSUE TITLE
[Feature] Remove left extra padding before menu items

### ISSUE THREAD

**Issue by enoversum** [LiveEnhancementSuite/LESforWindows#13]
On a previous version, there was a lot less padding on the left side of items in the right-click menu. Now the menu gets unnecessarily wide, and looks a little less polished than before, i.e. the padding on the top, right and bottom are still a lot slimmer. It’s just a small thing, but still a little improvement if this is being reverted to the style it had before 😘.

![IramaKJgWi](https://user-images.githubusercontent.com/8190174/135719323-8c91c596-a721-4b57-a621-72301c896f01.png)

**Comment by Inversil** [comment]
I'm pretty sure I haven't changed anything to how menus are handled inside AHK. you're on windows right? what's the theme you're using? looks good.

**Comment by enoversum** [comment]
@Inversil Yes, I am on Windows, and I am using a theme called [BIB3](https://www.deviantart.com/niivu/art/BIB3-for-Windows-886441919) by niivu, which is a full-black one. (Sometimes that leads to black-on-black, or white-on-white text in dialogue windows, the only downside.)
I just checked with a default Windows theme – same problem there. I remember it came with an update from you, maybe 1.30 or 1.32, it’s already too long ago. So it’s not the theme, but maybe some kind of Windows (11) setting that changed? I’m on Windows 11 Beta, but I believe this already appeared before the 11 Betas, and after I installed an update of yours.

Still not a breathtakingly annoying issue, so don’t lose hair over this 😄 .

**Comment by Inversil** [comment]
Hmm I hate to be that guy but there’s some big odds this is caused by a windows setting. 

I’ve had the menu do this for me but it’s currently not doing it; some tweak to windows in your registry caused the whitespace to show up, and the most likely candidate for that is windows 11, as AHK hasn’t changed the menu implementation for like 5 years; and I haven’t touched it since 1.2

i guess if anyone else sees this me know if you know what setting causes it; it’s probably pretty simple to fix but its specific to your system.

**Comment by enoversum** [comment]
@Inversil Thanks for getting back! Really strange, yet it could also have been 1.2 when the padding introduced itself (I don't remember which version it was and when exactly the padding occurred). 
However, I wouldn't swear it was your update, not unlikely that some Windows Insider change for those windows introduced it. If anyone else has an idea, or the same issue, maybe we'll find the culprit.
